_fellows/natasha-bansgopaul.md

```diff
@@ -7,14 +7,14 @@ project: []
 
-Natasha holds an MBA from The Pennsylvania State University (PSU), and has successfully led her companies to secure numerous awards globally such as, “Startup of the Year,” NextMoney Global FinTech Finals, HFM Technology “Best FinTech Solution for Hedge Funds,” and “Best Blockchain Technology Provider” in the 2019 from WealthManagement.com.
+Natasha Bansgopaul is a serial entrepreneur and technologist focusing on the intersection of emerging technologies and traditional infrastructure. She is the Co-Founder and COO of VegaX Holdings, an award-winning financial  technology platform building cryptocurrency index technology and automated portfolio rebalancing  infrastructure solutions for traditional institutions and investors. Natasha is an experienced financial technology, marketing, research, and M&A professional with more than 10 years of demonstrated  leadership through key strategic roles at Fortune100 companies.
 
-She has received notable recognition from prestigious organizations/publications, demonstrating her significant impact throughout financial technology. Of note, she was featured in FastCompany, Black Enterprise, Korea Times, Essence, and ThriveGlobal, in addition to receiving the distinguished honor of “Female Executive of the Year” and “Female Entrepreneur of the Year” in 2018. In 2020, Natasha was recognized as PSU’s 2020 Alumni Achievement Honoree, recognizing achievements from alumni 35 years and younger.
+Featured  in Forbes, Fortune, Black Enterprise, Korea Times, and Fast Company, Natasha was recognized as “Female Entrepreneur of the Year” in 2018 and has led her companies to secure numerous awards globally such as, NextMoney Global  FinTech Finals, “Best FinTech Solution for Hedge Funds,” and “Best Blockchain Technology Provider” in  the 2019 Wealth Management.com Industry Awards.
 
-With a passion for learning, business development, diversity and inclusion, financial literacy, and technology, Natasha continues to pioneer the need for change in the financial services industry via fintech solutions.
+Most recently, Natasha was named by Cosmopolitan and digitalundivided to the 2022 The New C-Suite, honoring women entrepreneurs who are heading the evolution of business leadership in America. Natasha graduated with her MBA from Pennsylvania State University, with a focus on entrepreneurship and strategic leadership, and her BS from Binghamton University. With a passion for learning, business  development, diversity and inclusion, and universal access to technology, Natasha continues to be a  leader and advocate for the development and adoption of FinTech and A.I. solutions globally.
```

Comment:
"digitalundivided" should be "DigitalUndivided" 
Add quotes: "The New C-Suite"
May be an extra space between "business" and "development" + "a" and "leader"